### PR TITLE
Make the plugin Java 11 ready -- Remove JAXB usage

### PR DESCRIPTION
**Pull request by batmat**
`mvn -V clean verify -Djenkins.version=2.164.1` works with this fix, both on a JDK8 and JDK11.

@jenkinsci/java11-support

### CODE REVIEW

**Comment by batmat** [comment]
infra issue...

**Comment by batmat** [comment]
@stephenc It is ready for your review, and hopefully approval & merge :). Thanks!

**Comment by batmat** [comment]
This will also fix https://issues.jenkins-ci.org/browse/JENKINS-56622

File: pom.xml [hunk 1]
```diff
@@ -67,8 +67,8 @@
   <properties>
     <revision>2.3.2</revision>
```
Comment (stephenc) [inline pom.xml]: 2.4.0

Comment (batmat) [reply pom.xml]: Done Stephen

**Comment by batmat** [comment]
@stephenc comment addressed, and conflict with `master` fixed.

**Comment by batmat** [comment]
Thanks Stephen!

**Comment by drdamour** [comment]
@stephenc i think the release notes on https://plugins.jenkins.io/scm-api have a typo they say required 2.160.3 not 2.60.3

**Comment by drdamour** [comment]
Screen Shot 2019-03-26 at 6 47 09 PM

**Comment by casz** [comment]
@drdamour anyone can fix that typo as long as you have a Jenkins account. https://accounts.jenkins.io/signup

On the plugins page to the right you should see the text: `ARE YOU MAINTAINING THIS PLUGIN?
Visit the Jenkins Plugin Wiki to edit this content.`
The link takes you to https://wiki.jenkins.io/display/JENKINS/SCM+API+Plugin

Where you can sign in with your Jenkins account

**Pull request merged**